packages/augur-ui/src/modules/common/constants.ts

```diff
@@ -683,6 +683,8 @@ export const WARNING = 'warning';
 export const ERROR = 'error';
 export const UP = 'up';
 export const DOWN = 'down';
+export const BETTING_LAY = 'betting lay';
```

Comment:
if these aren't used for text, we shouldn't use spaces in it. in other words if it's just for setting a type (back or lay) we should just have it match the const name `BETTING_LAY = 'BETTING_LAY';` kind of deal

Reply:
Updated. Thank you.